nixos/modules/services/networking/networkmanager.nix

```
@@ -337,6 +330,71 @@ in {
           so you don't need to to that yourself.
         '';
       };
+
+      enableFortiSSL = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable the FortiSSL plugin.
+          </para><para>
+          If you enable this option the
+          <literal>networkmanager_fortisslvpn</literal> plugin will be added to
+          the <option>networking.networkmanager.packages</option> option
+          so you don't need to to that yourself.
```

Maybe something like:

```suggestion
          If you enable this option, the
          <literal>networkmanager_fortisslvpn</literal> plugin will be added to
          <option>networking.networkmanager.packages</option> for you.
```

(mimic however you decide to reorganize this in all the other options as well)

---

Commited something similar, thank you!